piccolo/querystring.py

```diff
@@ -91,8 +97,10 @@ def __str__(self):
             if _type == str:
                 converted_args.append(f"'{arg}'")
             elif _type == datetime:
-                dt_string = arg.isoformat().replace("T", " ")
```

Comment:
I checked with and without the `'T'`, and both seem to work. So you're right, the `replace` is redundant.